back/db/migrate/20230718121501_create_followers.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateFollowers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :followers, id: :uuid do |t|
+      t.string :followable_type, null: false
+      t.uuid :followable_id, null: false
```

Comment:
```suggestion
      t.references :followable, polymorphic: true, null: false
```

We also need to make sure that the right indexes for `:followable` exist. (There are at least needed to run the dependent-destroy callbacks efficiently.)

Reply:
Which index is missing?